packages/stark-core/karma.conf.js

```diff
@@ -11,7 +11,20 @@ const karmaTypescriptFiles = [{ pattern: helpers.root("index.ts") }, { pattern:
 // start customizing the Karma configuration from stark-testing
 const starkCoreSpecificConfiguration = Object.assign({}, defaultKarmaConfig, {
 	// add missing files due to "@nationalbankbelgium/stark-core" imports used in mock files of the testing sub-package
-	files: [...defaultKarmaConfig.files, ...karmaTypescriptFiles]
+	files: [...defaultKarmaConfig.files, ...karmaTypescriptFiles],
+	// Overwrite the karmaTypescriptConfig to pass the correct preset to karma-typescript-es6-transform
+	karmaTypescriptConfig: {
+		...defaultKarmaConfig.karmaTypescriptConfig,
+		bundlerOptions: {
+			...defaultKarmaConfig.karmaTypescriptConfig.bundlerOptions,
+			transforms: [
```

Comment:
Can you add these to the exports of this file so that you can access it in `stark-core/karma.conf.ci.js` instead of duplicating this block in these 2 files?

Reply:
> I was thinking of doing this with #1145 (amongst other things). But I could append a commit to this PR.

Reply:
Mmm well I don't think that this will change that much when #1145 is solved right? We would need to apply the same logic here... so we could re-use it here instead of duplicating it

Reply:
I was thinking of writing (using) a merge function so that a single config file could be exported from the karma.conf.js